common/common.scss

```diff
@@ -1,214 +1,182 @@
+@import "badges_page";
```

Comment:
Not a big deal, but just a preference. I tend to use a relative path with a camelCase file name.

```scss
@import './badgesPage';
```

The relative path help removing conflicts/uncertainties if mix-using path alias/mappings.
Camel case is optimised for the global search comparing with snake_case. For example, when you search the styling file, it may save a character for `_`.

Reply:
This is due to discourse's limitations, see [this post](https://meta.discourse.org/t/splitting-up-theme-scss-into-multiple-files/115126). A relative path won't work.